src/Microsoft.Dnx.DesignTimeHost/ProjectStateResolver.cs

```diff
@@ -72,9 +78,18 @@ public ProjectState Resolve(string appPath, string configuration, bool triggerBu
 
             var sourcesProjectWidesources = project.Files.SourceFiles.ToList();
 
+            ResolveSearchPaths(state);
+
+            var searchPathUpdated = currentSearchPaths != null &&
```

Comment:
Explaination: `currentSearchPath` is `null` when the project hasn't been initialized, which means it is processing the `Initialization` request now. There is no sense to validate the dependencies validation under changed search paths in this situation.